### PR TITLE
Added sessions table to exclude list for exporter

### PR DESCRIPTION
**Pull request by naz**
refs #9865

- Added sessions table to be skipped during the export process

### CODE REVIEW

File: core/test/unit/data/exporter/index_spec.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ describe('Exporter', function () {
             // Execute
             exporter.doExport().then(function (exportData) {
                 // No tables, less the number of excluded tables
-                var expectedCallCount = schemaTables.length - 4;
+                var expectedCallCount = schemaTables.length - exporter.EXCLUDED_TABLES.length;
```
Comment (gargol) [inline core/test/unit/data/exporter/index_spec.js]: Just gotten rid of this magic number as needed to fix the test case itself anyway

File: core/test/unit/data/exporter/index_spec.js [hunk 2]
```diff
@@ -94,8 +94,8 @@ describe('Exporter', function () {
 
             // Execute
             exporter.doExport({include: ['clients', 'client_trusted_domains']}).then(function (exportData) {
-                // all tables, except of the tokens
-                const expectedCallCount = schemaTables.length - 2;
+                // all tables, except of the tokes and sessions
+                const expectedCallCount = schemaTables.length - (exporter.EXCLUDED_TABLES.length - 2);
```
Comment (gargol) [inline core/test/unit/data/exporter/index_spec.js]: Same as above, and here the 2 comes from - `['clients', 'client_trusted_domains']`

**Review by allouis — APPROVED** [review]
:+1:

File: core/server/data/exporter/index.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ var _ = require('lodash'),
     common = require('../../lib/common'),
     security = require('../../lib/security'),
     models = require('../../models'),
-    EXCLUDED_TABLES = ['accesstokens', 'refreshtokens', 'clients', 'client_trusted_domains'],
+    EXCLUDED_TABLES = ['accesstokens', 'refreshtokens', 'clients', 'client_trusted_domains', 'sessions'],
```
Comment (kirrg001) [inline core/server/data/exporter/index.js]: @allouis What about api keys? Should they be excluded?

And in general: what about importing/exporting integerations?

Comment (kirrg001) [reply core/server/data/exporter/index.js]: > Is it mainly used for transferring a blog to another server?

Yeah and backup for the same blog.

> If so, you might want all your integrations to keep working? 

Yeah. Just need to think about the security risk of exporting admin keys.

**Comment by kirrg001** [comment]
Can this be merged?

**Comment by gargol** [comment]
@kirrg001 depends if we want to do any more changes regarding https://github.com/TryGhost/Ghost/pull/9961#discussion_r225079453 ?

**Comment by kirrg001** [comment]
I'd suggest to merge this and @allouis could keep track of this. We currently don't allow to create an admin key right? Because we only support content keys.

**Pull request merged**